### PR TITLE
Update index.rst

### PR DESCRIPTION
**Pull request by JonathanDensil**
Added links to pep8 and pycodestyle as described in Issue #602 by @IanLee1521.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -5,7 +5,7 @@ pep8's documentation
 
 .. note::
 
-    This package used to be called ``pep8`` but was renamed to ``pycodestyle``
+    This package used to be called `pep8 <https://www.python.org/dev/peps/pep-0008/>`_ but was renamed to `pycodestyle <https://pypi.python.org/pypi/pep8>`_
```
Comment (sigmavirus24) [inline docs/index.rst]: There's a few things wrong with this:

1. There was a maximum line length being observed in this document that is being violated by this line
2. The first link should not be to the document PEP-0008, but instead to the package on PyPI pep8.
3. The second link should not be to the package on PyPI named "pep8" but to the one named "pycodestyle"

Comment (JonathanDensil) [inline docs/index.rst]: Oh right, my apologies, I now understand the issue. I will rectify this immediately.

**Comment by JonathanDensil** [comment]
Hi @sigmavirus24  and @IanLee1521, I corrected the problem with the previous commit, changed the link addresses and abided by the line length guide as requested by sigmavirus24 in his review of my previous commit. I hope that I have done it right this time. Thank you, please review.

**Comment by sigmavirus24** [comment]
The test failures were unrelated.

**Comment by JonathanDensil** [comment]
Thank you very much @sigmavirus24 and @IanLee1521, it was a very exciting experience. I hope to make more useful contributions in the future!

**Pull request merged**